src/router.js

```diff
@@ -33,11 +33,10 @@ const routes = [
 	{ path: '/config', component: Config, props: false },
 	{ path: '/userinfo', component: UserInfo, props: false, meta: { auth: true } },
 	{ path: '/dataset/:id', name: "editor", component: Editor, props: true, meta: { auth: true }, children: [
-		{ path: '', redirect: { name: 'tab' } },
```

Comment:
why we remove this, should we just redirect to that tab but with a default value for the tab property?

Reply:
The tabs come from the active UI schema, which doesn't exist until we open a dataset with a schema in the Editor. To have a default tab, we have to assume that all datasets have a specific tab, which is true with the two current schemas, but not for new datasets. A new dataset doesn't initially have a schema so no tabs actually exist at that point.

Reply:
ok